_pages/story.md

```diff
@@ -9,15 +9,29 @@ We would like other agencies, developers, and groups to use eRegulations and ada
 
 # History
 
-[CFPB released its initial version of eRegulations in 2013](http://www.consumerfinance.gov/blog/making-regulations-easier-to-use/), and [18F and ATF began reusing it for ATF in 2015](https://18f.gsa.gov/2015/12/09/an-open-source-government-is-a-faster-more-efficient-government/).
+[CFPB released its initial version of eRegulations in 2013](http://www.consumerfinance.gov/blog/making-regulations-easier-to-use/), and [18F and ATF began reusing it for ATF in 2015](https://18f.gsa.gov/2015/12/09/an-open-source-government-is-a-faster-more-efficient-government/). 
```

Comment:
Accidental space added... I was thinking about explaining FEC and N&C here as part of eRegs' history because the new news clippings below this talks about that. But, I'm not quite sure how to restructure this page at the moment.